third_party/ui/bower_components/angular-material/modules/closure/select/select.js

```diff
@@ -757,7 +757,7 @@ function SelectProvider($$interimElementProvider) {
       var focusedNode = centeredNode || optionNodes[0];
       if (focusedNode) {
         opts.focusedNode = focusedNode;
-        focusedNode.focus();
+        // focusedNode.focus();
```

Comment:
Please convert it to an explanatory comment rather than leaving it as commented-out code.